release-calendar/src/server/repository-service.ts

```diff
@@ -57,4 +57,58 @@ export class RepositoryService {
 
     return releaseQuery;
   }
+
+  include(arr: Release[], rel: Release): boolean {
```

Comment:
You could use this instead - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
The function was supposes to check if the `rel` releases was included in `arr` of releases. I should have named the function `isIncluded`. Anyway, I did poke around in javascript build in functions and switched to `.indexOf()` because if `.indexOf()` returns `-1` then the element was not found in the array.